e2e/test/scenarios/admin/i18n/content-translation/column-names-in-viz.cy.spec.ts

```diff
@@ -0,0 +1,289 @@
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import type { StructuredQuestionDetails } from "e2e/support/helpers";
+import { type CardDisplayType, cardDisplayTypes } from "metabase-types/api";
+
+import { germanFieldNames } from "./constants";
+import {
+  interceptContentTranslationRoutes,
+  uploadTranslationDictionary,
+} from "./helpers/e2e-content-translation-helpers";
+const { PEOPLE, PEOPLE_ID, PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const { H } = cy;
+
+describe("scenarios > admin > localization > content translation of column names in static-embedded visualizations", () => {
+  describe("ee", () => {
+    beforeEach(() => {
+      interceptContentTranslationRoutes();
+    });
+
+    before(() => {
+      H.restore();
```

Comment:
I think we generally do a restore between each test so that we always start from a known place. Was that too slow to do in this case? Would uploading the translation dictionary via API help speed it up if so?